lua/wire/zvm/zvm_tests.lua

```diff
@@ -0,0 +1,411 @@
+--------------------------------------------------------------------------------
+-- Library for making and running automated tests for Zyelios Virtual Machine Programs
+--
+--
+--------------------------------------------------------------------------------
+
+TESTING = true
+include("wire/cpulib.lua")
+include("wire/client/hlzasm/hc_compiler.lua")
+
+
+ZVMTestSuite = {
+	TestFiles = {},
+	TestQueue = {},
+	TestStatuses = {},
+	Warnings = 0,
+	CurrentWarnings = 0
+}
+
+local testDirectory = "wire/zvm/tests"
+
+function ZVMTestSuite.CMDRun(_,_,_,names)
+	ZVMTestSuite.Warnings = 0
+	ZVMTestSuite.TestFiles = {}
+	for filename in string.gmatch(names, '[^,]+') do
+		local files = file.Find('lua/'..testDirectory..'/'..filename..'.lua',"GAME")
```

Comment:
Why would there ever be multiple files returned here? Simplify this to just `file.Read`?

Reply:
I split on commas, so you can type "test1,test2" to run test1.lua and test2.lua
file.Find seems to allow * which I don't strip from the string, so you can type "*example*" to run every test with the word "example" in it (file_example.lua, example.lua)

So you can debug with a few tests at a time rather than all of them at once if you specify the names in a comma delimited list.